src/shared/components/Popover/Popover.tsx

```diff
@@ -53,9 +55,11 @@ export const Popover: React.FC<PopoverProps> = ({
         })
       }}
       render={(attrs) => (
-        <ContentContainer {...attrs} className={className}>
-          {content}
-        </ContentContainer>
+        <div ref={contentContainerRef}>
+          <ContentContainer {...attrs} className={className}>
+            {content}
+          </ContentContainer>
+        </div>
```

Comment:
Why not apply `ref` directly to `ContentContainer`?

Reply:
The transform css animation mess up the meassuring